lib/github-auth.js

```diff
@@ -2,14 +2,15 @@ var querystring = require('querystring');
 var request = require('request');
 var autosave = require('json-autosave');
 var serverSecrets;
+var baseUrl = process.env.BASE_URL || "https://shields.io";
```

Comment:
Just tested that locally, this results in a redirect loop on the homepage, because it matches `BASE_URL`, but also redirects to `BASE_URL`. My suggestion is to keep the distinction of `BASE_URL` and `INFOSITE` anyway.